libs/components/modals/src/lib/modules/modal/modal-footer.component.scss

```diff
@@ -2,6 +2,9 @@
 @use 'libs/components/theme/src/lib/styles/variables' as *;
 
 .sky-modal-footer-container {
+  --sky-adjacent-btn-margin: #{$sky-margin};
```

Comment:
You can get rid of these custom properties when you apply the suggestions in the other comment.